src/liquid/components/ld-toggle/readme.md

```diff
@@ -98,6 +118,12 @@ This component can be used in conjunction with the [`ld-label`](components/ld-la
 
 <ld-toggle aria-disabled="true" checked></ld-toggle>
 
+<!-- React component -->
+
+<LdToggle ariaDisabled="true" />
+
+<LdToggle ariaDisabled="true" checked />
```

Comment:
```suggestion
<LdToggle aria-disabled="true" />

<LdToggle aria-disabled="true" checked />
```

These seem to be the only two examples, where you used camelCase for aria-attributes. All other examples (also Boris and my ones) use kebab-case also for React examples. Even though this may be a bit inconsistent to non-aria-attributes, we should keep it uniform. Apart from this, I assume that @borisdiakur  already reviewed the whole thing, that's why I'm giving my approval.

Reply:
Good catch. Yes, apart from this, I also approve 🦭